scripts/docker-serve-full.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+# navigate to the repository root
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )/.."
+
+JEKYLL_DOCKER_IMAGE_VERSION=pages-v228-minimal
```

Comment:
Does this container not provide multiarch? https://github.com/duckdb/duckdb-web/blob/master/.devcontainer/devcontainer.json#L6C12-L6C61

Reply:
It does, but it's built-in Jekyll is too new: v4.3.2. So we would need to reinstall Jekyll to match the one used by us (3.9.2).